opensciencegrid/gwms-factory/etc/osg/image-init.d/11_fix_uids.sh

```diff
@@ -6,7 +6,7 @@ if [ $(stat -c %U:%G /var/lib/condor/spool) != "condor:condor" ] ; then
   chown -R condor:condor /var/lib/condor/*
 fi
 
-if [ $(stat -c %U:%G /var/lib/gwms-factory/work-dir) != "gfactory:gfactory" ] ; then
+if [ $(stat -c %U:%G /var/lib/gwms-factory/web-area/stage) != "gfactory:gfactory" ] ; then
   chown -R gfactory:gfactory /var/lib/gwms-factory/*
 fi
```

Comment:
Would it be OK to look at each subdirectories, in case the chown is interrupted?
```suggestion
for entry in /var/lib/gwms-factory/* ; do
  if [ $(stat -c %U:%G "$entry") != "gfactory:gfactory" ] ; then
    chown -R gfactory:gfactory "$entry"
  fi
done
```